k8s/templates/puppetdb-deployment.yaml

```diff
@@ -27,12 +27,16 @@ spec:
           resources:
             {{- toYaml .Values.puppetdb.resources | nindent 12 }}
           env:
-            {{- range $key, $value := .Values.puppetdb.extraEnv }}
+          {{- range $key, $value := .Values.puppetdb.extraEnv }}
             - name: {{ $key }}
               value: {{ $value }}
-            {{- end }}
+          {{- end }}
             - name: PUPPETSERVER_HOSTNAME
               value: "puppet"
+          {{- if .Values.puppetserver.preGeneratedCertsJob.enabled }}
+            - name: USE_PUPPETSERVER
+              value: "false"
```

Comment:
You shouldn't need to set this unless certs aren't getting mounted into the puppetdb container correctly, see code in https://github.com/puppetlabs/puppetdb/blob/master/docker/puppetdb/docker-entrypoint.d/30-configure-ssl.sh

Reply:
@underscorgan Do we really need the same certs for PuppetDB - if we have these for Puppetserver?

Reply:
It won't be the same mount from the puppetserver, but you'll need the certs the from the puppetdb bare-metal machine, at least `${SSLDIR}/certs/${CERTNAME}.pem` and `${SSLDIR}/private_keys/${CERTNAME}.pem`. If you have the certs loaded on the puppetserver and don't have the certs on puppetdb, things will start up but the connection between puppetdb and puppetserver won't work.

Reply:
Yup - confirmed. Thanks! 
That will make things even more complex but we've got that far - so let's finish it.